server/src/controller/item.ts

```diff
@@ -53,18 +66,28 @@ export async function createItem(
       await queryRunner.manager.save(itemTag);
     }
 
+    for (let i = 0; i < picturesArray.length; i += 1) {
```

Comment:
We need to validate for 409 error, when trying to create an item with the same supplier and name getting a 500 error